.travis/publish.sh

```diff
@@ -1,11 +1,22 @@
 #!/usr/bin/env bash
 
+set -e
+
 if [ ! -z "$TRAVIS_TAG" ]
 then
-    echo "deploying beadledom $TRAVIS_TAG to maven central"
+    echo "Decrypting secrets ..."
+    openssl aes-256-cbc -K $encrypted_4d3aca009c62_key -iv $encrypted_4d3aca009c62_iv -in $GPG_DIR/deploy_site_key.enc -out $GPG_DIR/deploy_site_key -d
+    openssl aes-256-cbc -pass pass:$ENCRYPTION_PASSWORD -in $GPG_DIR/pubring.gpg.enc -out $GPG_DIR/pubring.gpg -d
+    openssl aes-256-cbc -pass pass:$ENCRYPTION_PASSWORD -in $GPG_DIR/secring.gpg.enc -out $GPG_DIR/secring.gpg -d;
+
+    echo "git checking out $TRAVIS_TAG"
     git checkout tags/$TRAVIS_TAG -b tag_$TRAVIS_TAG
```

Comment:
technically speaking travis doesn't  checkout the tag. It checksout the master and switches to a specific commit at which the tag was cut.  I guess they both resolve to the same command internally. Removed checking  out tag explicitly.